drivers/disability_summary/app/views/disability_summary/warehouse_reports/disability_summary/index.haml

```diff
@@ -0,0 +1,76 @@
+:ruby
+  reports = [@report]
+  column_class = 'col-sm-8'
+  if @report.include_comparison?
+    reports << @comparison
+    column_class = 'col-sm-6'
+  end
+  @layout__plain_inner = true
+  @layout__width = 'lg'
+
+- title = _('Disability Summary Report')
+- content_for :title, title
+.content-outer-prs1
+  %nav.nav-drawer.prs1
+    = render 'filters/filter_controls/sidebar'
+  .main-outer-prs1
+    .utility-prs1
+      = render '/warehouse_reports/breadcrumbs', tab: 'operational'
+      = render 'utility'
+    .top-nav-prs1
+      %h1.mb-5= title
+      .mb-4= render 'hero_counts'
+      .main-inner-prs1
+        - if false
```

Comment:
I assume this is disabled intentionally? Maybe put in a comment to that effect, and call out that the implementation is incomplete?

Reply:
Good catch